cmd/connector-common-winners-timescale/main.go

```diff
@@ -5,17 +5,79 @@
 package main
 
 import (
+	"math"
+	"math/big"
+
+	"github.com/fluidity-money/fluidity-app/lib/log"
 	"github.com/fluidity-money/fluidity-app/lib/databases/postgres/solana"
+	"github.com/fluidity-money/fluidity-app/lib/databases/timescale/lootboxes"
 	database "github.com/fluidity-money/fluidity-app/lib/databases/timescale/winners"
 	queue "github.com/fluidity-money/fluidity-app/lib/queues/winners"
+	"github.com/fluidity-money/fluidity-app/lib/types/network"
 )
 
+func lootboxUpdateTrackedRewardAmounts(winner queue.Winner) {
```

Comment:
add a comment to this function? also i think there was an existing microservice that handled this separately from winners-timescale

Reply:
I couldn't find a suitable existing service, so I tacked it onto the winners connector, which seems fine to me.